schemata/cta.yaml

```diff
@@ -10,7 +10,11 @@ properties:
   id:
     "$ref": "base.json#/definitions/uuid"
   type:
-    "$ref": "contentblock.json#/definitions/type"
+    description: Block type
+    example: "text"
+    type: string
```

Comment:
OK, trying to understand here these changes to `type`: `contentblock` still defines what are the possible types of blocks, however each block (e.g. `cta`) should define explicitly its type instead of depending on any kind of type from `contentblock`. Right?

Does it means that before this change a `cta` block could has any type like `h1`?